android/app/src/main/java/com/tourneynizer/tourneynizer/MainActivity.java

```diff
@@ -0,0 +1,28 @@
+package com.tourneynizer.tourneynizer;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.text.Editable;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+public class MainActivity extends AppCompatActivity {

```

Comment:
Is there a reason that you chose to extend AppCompatActivity here?

Reply:
This was auto generated. I believe that it has something to do with the action bar at the top of the UI. The android API said that with AppCompatActivity, I can add functionality to the action bar. For now I'll just use a regular Activity, but may return to it later for added functionality.

Reply:
Turns out that without AppCompatActivity, the action bar dissapears so I am going to add it back.